### PR TITLE
[PhpUnitBridge] Lint files with PHP 5.5

### PR DESCRIPTION
**Pull request by derrabus**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | N/A
| License       | MIT
| Doc PR        | N/A

I'd like to make sure that PhpUnitBridge remains compatible with PHP 5.5.

cc @nicolas-grekas @fancyweb

### CODE REVIEW

File: .github/workflows/phpunit-bridge.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    coverage: "none"
+                    php-version: "5.5"
+            -   name: Lint
+                run: find ./src/Symfony/Bridge/PhpUnit -name '*.php' | grep -v -e /Tests/ -e ForV6 -e ForV7 -e ForV8 -e ForV9 -e ConstraintLogicTrait | parallel -j 4 php -l {}
```
Comment (OskarStark) [inline .github/workflows/phpunit-bridge.yml]:
```suggestion
            -   name: Checkout
                uses: actions/checkout@v2

            -   name: Setup PHP
                uses: shivammathur/setup-php@v2
                with:
                    coverage: "none"
                    php-version: "5.5"

            -   name: Lint
                run: find ./src/Symfony/Bridge/PhpUnit -name '*.php' | grep -v -e /Tests/ -e ForV6 -e ForV7 -e ForV8 -e ForV9 -e ConstraintLogicTrait | parallel -j 4 php -l {}
```

**Comment by nicolas-grekas** [comment]
Could we even run the tests with 5.5?
(and remove the corresponding logic from .travis.yml)
Just: please make this a single job on GHA :)

**Comment by derrabus** [comment]
> Could we even run the tests with 5.5?

I tried last night and failed miserably. 🙈

**Comment by derrabus** [comment]
@nicolas-grekas Can we merge this PR as is? The tests are widely incompatible with php 5.5 and this small linter (that only activates on changes to the bridge) would have been enough to detect the compatibility issues we've had recently.

**Review by nicolas-grekas — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @derrabus.

**Comment by derrabus** [comment]
After merging up, I've removed the new linter from the 5.x branch again. It doesn't make much sense there. :-)

**Pull request merged**